src/modules/access/access/buildAccessDetails.tsx

```diff
@@ -111,11 +126,17 @@ export function buildAccessDetails({
       }),
```

Comment:
A "restrict" field with value `0x0000000000000000000000000000000000000000` means "not restricted"

This could be enhanced:
- should we display the line anyway if the value is `0x0000000000000000000000000000000000000000`?
- or simply add "no restriction" in the label? 🤔